include/fmt/format.h

```diff
@@ -1460,7 +1460,7 @@ FMT_CONSTEXPR unsigned parse_nonnegative_int(
   }
   unsigned value = 0;
   // Convert to unsigned to prevent a warning.
-  unsigned max_int = (std::numeric_limits<int>::max)();
+  unsigned max_int = internal::to_unsigned(std::numeric_limits<int>::max());
```

Comment:
This fails to build on Windows. Somehow I get a feeling we're bitten by some max() macro coming from system headers or from the infamous <windows.h>

Ugh... so that's why it looked like that...

https://stackoverflow.com/questions/11544073/how-do-i-deal-with-the-max-macro-in-windows-h-colliding-with-max-in-std